tests/handlers/test_user_directory.py

```diff
@@ -56,6 +56,7 @@ class UserDirectoryTestCase(unittest.HomeserverTestCase):
 
     def make_homeserver(self, reactor: MemoryReactor, clock: Clock) -> HomeServer:
         config = self.default_config()
+        # Always process user directory updates on this main homeserver process.
```

Comment:
this isn't terribly clear.
```suggestion
        # Enable user directory updates
```

Reply:
Yes, but this comment exists to explain *why* the user updates flag is set to true there.